src/Microsoft.Diagnostics.ExtensionCommands/DumpAsyncCommand.cs

```diff
@@ -655,37 +661,42 @@ bool IncludeInOutput(ClrObject obj)
             // </remarks>
             void AddContinuation(ClrObject continuation, List<ClrObject> continuations)
             {
-                if (continuation.Type.Name.StartsWith("System.Collections.Generic.List<", StringComparison.Ordinal))
+                if (continuation.Type is not null)
                 {
-                    if (continuation.Type.GetFieldByName("_items") is ClrInstanceField itemsField)
+                    if (continuation.Type.Name is not null &&
```

Comment:
Combine these two if's: 
```suggestion
                    if (continuation?.Type?.Name is not null &&
```